tests/test_kms.py

```diff
@@ -39,4 +39,36 @@ def test_key_rotation(self):
             session_factory=session_factory)
 
         resources = p.run()
-        self.assertEqual(len(resources), 7)
+        self.assertEqual(len(resources), 2)
+
+    def test_kms(self):
```

Comment:
what is this test asserting?.. afaics its just doing standard value semantics on kms keys, its unclear that its adding value as a result. if it is please add a comment to what its actually testing or make that more explicit.